### PR TITLE
playwares.com - Ad, sign.dcinside.com - Leftover

### PR DESCRIPTION
**Pull request by green1052**
http://playwares.com/gametalk/43849046

### CODE REVIEW

File: filters-share/specific_CSS.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ loawa.com#$#div[id*="ad_"][style*="position: absolute; top:"] { top: 100% !impor
 loawa.com#$#div[class*="p-0 m-0 mb-2 d-none d-md-block text-center "] { position: relative !important; top: -200px !important; }
 loawa.com#$##ad_center_v.adsense-ch { position: relative !important; top: 420px !important; }
 !
+sign.dcinside.com#$#.login_inputbox .inner { width: calc(100% - 115px) !important; }
```
Comment (piquark6046) [inline filters-share/specific_CSS.txt:8]: Does this rule make dcinside login page looks like better?

Comment (green1052) [reply filters-share/specific_CSS.txt:8]: <details>
<summary>Screenshot:</summary>

imagee5596a6ec69e.png)

</details>
@piquark6046

Comment (piquark6046) [reply filters-share/specific_CSS.txt:8]: Did you test the rule on a chromium browser?
@green1052

Comment (green1052) [reply filters-share/specific_CSS.txt:8]: <details>
<summary>Screenshot:</summary>

image

</details>

with two rules
```
sign.dcinside.com#$#.login_inputbox .inner { width: calc(100% - 115px) !important; }
sign.dcinside.com##.ban_box
```

Comment (green1052) [reply filters-share/specific_CSS.txt:8]: > Did you test the rule on a chromium browser?

yeah i tested in ungoogled chromium

Comment (piquark6046) [reply filters-share/specific_CSS.txt:8]: @green1052 
With AdGuard for Windows?

Comment (green1052) [reply filters-share/specific_CSS.txt:8]: nop ublock

Comment (piquark6046) [reply filters-share/specific_CSS.txt:8]: Please test new rule on a chromium browser.

Comment (green1052) [reply filters-share/specific_CSS.txt:8]: sadly i turn off my pc

**Comment by green1052** [comment]
i did sync if sytle

File: filters-share/specific_CSS.txt (hunk 1 above)
Comment (green1052) [reply filters-share/specific_CSS.txt:8]: Screenshot_20220529-004406_Kiwi Browser

Screenshot_20220529-004417_AdGuard

Comment (green1052) [reply filters-share/specific_CSS.txt:8]: @piquark6046

Comment (piquark6046) [reply filters-share/specific_CSS.txt:8]: No. You have to add the following rules when you test on a chromium browser:
```
sign.dcinside.com#$#.login_inputbox { width: 100% !important; }
sign.dcinside.com#$#.login_inputbox .inner { width: calc(100% - 115px) !important; }
sign.dcinside.com##.ban_box
```

**Review by green1052 — COMMENTED** [review]
https://github.com/List-KR/List-KR/blob/4698b8fbc618d5be3a49e69373e8d660b764974b/filter.txt#L20

File: filters-share/specific_CSS.txt (hunk 1 above)
Comment (green1052) [reply filters-share/specific_CSS.txt:8]: Screenshot_20220529-004924_Kiwi Browser

its working sorry

**Comment by piquark6046** [comment]
https://github.com/List-KR/List-KR/pull/374#pullrequestreview-988533522
Oh, I made a mistake.
Sorry about that.

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**